src/interaction/DihedralHarmonic.hpp

```diff
@@ -187,13 +199,19 @@ namespace espressopp {
        *
        * @return The value of the force
        */
+
+      //TODO note: this function has not been tested and may need debugging
       real _computeForceRaw(real phi) const {
+        std::cout<<"Warning! The function _computeForceRaw(real phi) in DihedralHarmonic has not been tested and may need debugging"<<std::endl;
```

Comment:
As far as I know, these _computeForceRaw(real) functions at the end of the most of the interaction potential files were intended for being called directly from python, e.g. if someone wanted to output a tabulated potential.

I never used any of them and don't know if anyone currently does... That's why I added the warning.